Wrappers/Python/test/test_reconstructors.py

```diff
@@ -284,7 +287,43 @@ def test_set_filter_inplace(self):
         with self.assertRaises(TypeError):
             reconstructor.set_filter_inplace('unsupported_value')
 
+    def create_custom_filter_example(self, cutoff):
+        """Returns a custom filter array."""
+        filter_length = 256
+        freq = fftfreq(filter_length)
+        freq*=2
+        ramp = abs(freq)
+        ramp[ramp>cutoff]=0
+        FBP_filter = ramp*(np.cos(freq*np.pi*4)+1*np.cos(1/5*freq*np.pi/2))/2
+        return FBP_filter
 
+    @unittest.skipUnless(has_tigre and has_ipp, "TIGRE or IPP not installed")
+    def test_plot_filter(self):
+        """
+        Tests that the filters are plotted correctly for two different 
+        values of cutoff, all preset filters and the custom filter.
+        The plots are compared to stored png files.
+        The test will not show any screen output.
+        """
+        fdk = GenericFilteredBackProjection(self.ad3D)
+        filter_list = fdk.preset_filters
+        filter_list.append('custom')
+        filter_plots_folder = r"../test_plots/filters"
+        test_plot_path = os.path.join(filter_plots_folder, 'test_plot_filter.png')
```

Comment:
It looks like one of the tests failed because it couldn't access this location to save the file
![image](https://github.com/TomographicImaging/CIL/assets/77114597/52ac7c1b-8ade-4b72-a24a-c41cefd5da55)
I had this problem and ended up using the cil data directory. That might be the best place for this as well
`data_dir = os.path.abspath(os.path.join(sys.prefix, 'share','cil'))`

Reply:
I edited the folder path, thank you for pointing this out. The tests are now passing locally as well as on conda